op-node/node/config.go

```diff
@@ -150,5 +160,16 @@ func (cfg *Config) Check() error {
 	if !(cfg.RollupHalt == "" || cfg.RollupHalt == "major" || cfg.RollupHalt == "minor" || cfg.RollupHalt == "patch") {
 		return fmt.Errorf("invalid rollup halting option: %q", cfg.RollupHalt)
 	}
+	if cfg.ConductorEnabled {
+		if state, _ := cfg.ConfigPersistence.SequencerState(); state != StateUnset {
+			return fmt.Errorf("config persistence must be disabled when conductor is enabled")
+		}
+		if !cfg.Driver.SequencerEnabled {
+			return fmt.Errorf("sequencer must be enabled when conductor is enabled")
+		}
+		if !cfg.Driver.SequencerStopped {
+			return fmt.Errorf("sequencer must be stopped when conductor is enabled")
+		}
```

Comment:
I think the two other checks above make sense, but I wonder if enabling the conductor should just automatically stop the sequencer given its job is to control whether the sequencer is started or not.  Avoids requiring the user to add pointless boilerplate options to their command.

Reply:
good point, resolved.